modules/compliance-filtering-failed-results.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+[id="filtering-failed-compliance-check-results_{context}"]
+= Filtering failed compliance check results
+
+By default, the `ComplianceCheckResult` objects are labeled with several useful labels that allow you to query the checks and decide on the next steps after the results are generated.
+
+
+To list checks that belong to a specific suite, run:
```

Comment:
s/To list checks that belong to a specific suite, run:/List checks that belong to a specific suite:/

Similar rewording suggestion applies to the other commands in this module (for minimalism and consistency across the Compliance Operator book).